packages/dds/tree/src/simple-tree/core/unhydratedFlexTree.ts

```diff
@@ -326,73 +308,81 @@ const unparentedLocation: LocationInField = {
 	index: -1,
 };
 
-class UnhydratedFlexTreeField implements FlexTreeField {
+export class UnhydratedFlexTreeField
+	implements FlexTreeField, MapTreeFieldViewGeneric<UnhydratedFlexTreeNode>
+{
 	public [flexTreeMarker] = FlexTreeEntityKind.Field as const;
 
-	public get context(): FlexTreeContext {
-		return this.simpleContext.flexContext;
-	}
+	public parent: UnhydratedFlexTreeNode | undefined = undefined;
 
 	public constructor(
-		public readonly simpleContext: Context,
+		public readonly context: FlexTreeContext,
```

Comment:
Docs?

Reply:
This is part of an interface implementation and thus implicitly gets inherited documentation.